server/api/static.js

```diff
@@ -0,0 +1,41 @@
+import { readFile } from 'fs'
+import path from 'path'
+import { URL } from 'node:url'
+
+const CLIENT_DIR = '../../client'
+const DIST_DIR = '/dist'
+const ASSETS_DIR = '/assets'
+
+const MIME_TYPES = {
+  '.js': 'text/javascript',
+  '.css': 'text/css',
+  '.json': 'application/json',
+  '.png': 'image/png',
+  '.jpg': 'image/jpg',
+  '.svg': 'image/svg+xml'
+}
+
+export default async function handleStatic(req, res) {
+  let filePath
+
+  if (req.url.includes(ASSETS_DIR)) {
+    filePath = new URL(`${CLIENT_DIR}${req.url}`, import.meta.url)
+  } else {
+    filePath = new URL(`${CLIENT_DIR}${DIST_DIR}${req.url}`, import.meta.url)
+  }
+
+  let extname = path.extname(req.url)
+
+  readFile(filePath, (error, content) => {
+    if (!error) {
+      res.writeHead(200, {
+        'Content-Type': MIME_TYPES[extname] || 'application/octet-stream',
+        'Cache-Control': 'public, max-age=31536000, immutable'
```

Comment:
In this PR you are serving some static assets without cache busters https://github.com/browserslist/browserl.ist/pull/324/files?short_path=be3af13#diff-edfa4829c58ad696e48ab606527eabead1471963a743486a94b65fe215035814R12

You can’t use `immutable` for them.

Be careful, think about the whole picture.

Reply:
Yes, I just wanted to add examples of static files without cache buster before dealing with `Cache-control` headers.

That's why I got distracted by that PR :)